code/modules/power/apc/apc_tool_act.dm

```diff
@@ -1,6 +1,21 @@
 //attack with an item - open/close cover, insert cell, or (un)lock interface
 /obj/machinery/power/apc/crowbar_act(mob/user, obj/item/crowbar)
 	. = TRUE
+
+		//Prying off broken cover
+	if(opened == APC_COVER_CLOSED || opened == APC_COVER_OPENED && (machine_stat & BROKEN))
+		crowbar.play_tool_sound(src)
+		user.visible_message(span_notice("[user.name] begins prying off the broken [name] cover!"))
+		balloon_alert(user, "you begin prying")
+		if(!crowbar.use_tool(src, user, 50))
+			return
+		opened = APC_COVER_REMOVED
+		user.visible_message(span_notice("[user.name] prys off the broken [name] cover!"))
+		balloon_alert(user, "the cover falls apart")
```

Comment:
you don't need both, stick to one. i think this balloon alert works better since "falls apart" implies something really bad happened to the integrity of the cover rather than simply removing it.

```suggestion
		balloon_alert(user, "cover removed")
```

Reply:
the first span is to tell surrounding people what the person is doing.

The falls apart is to show that something really bad did happen to the cover , its damaged beyond repair and fragmented, similar to how when you take the apc electronics out itl either break apart or you discard it if its malf`d or emagged